extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/tunnel/AbstractGuacamoleTunnelService.java

```diff
@@ -714,4 +714,15 @@ public GuacamoleTunnel getGuacamoleTunnel(RemoteAuthenticatedUser user,
 
     }
 
+
+    public Integer getIdleConnections(Connection connection) {
+    	Integer idle = 999999;
+        // Set parameters from associated data
+        Collection<ConnectionParameterModel> parameters = connectionParameterMapper.select(connection.getIdentifier());
+        for (ConnectionParameterModel parameter : parameters)
+        	if (parameter.getName().equalsIgnoreCase("balancing"))
```

Comment:
I mentioned this earlier, but I'll repeat here for the sake of context: this really is an abuse of connection parameters, which have a specific meaning.

It's also a little odd (or paranoid?) to use `equalsIgnoreCase()` here. If we *were* to define a magic parameter, better that it be strict. As written, this would be the only case where a parameter was interpreted in a case-insensitive way. Throughout the rest of the Guacamole stack, configuration options, parameters, attributes, properties, etc. are all case-sensitive and strictly interpreted.